packages/core/src/Stores/Helpers/client-notifications.js

```diff
@@ -299,6 +299,20 @@ export const clientNotifications = (ui = {}) => {
             message: localize('Your proof of identity document has expired. Please submit a new one.'),
             type: 'danger',
         },
+        new_version_available: {
+            action: {
+                onClick: () => location.reload(),
```

Comment:
For easier searches, can we prefix any reference to the global `location` with `window.`?
```suggestion
                onClick: () => window.location.reload(),
```

Reply:
Yep. Will do right away.